src/services.ts

```diff
@@ -31,6 +32,19 @@ export async function initialize (overrides: IEditorOverrideServices, container:
   initializeWorkbench(container, configuration)
 
   const instantiationService = StandaloneServices.initialize({
+    [IProductService.toString()]: Object.assign({
```

Comment:
what is the difference between `Object.assign` and `mixin`?

Reply:
No difference if there's no additional param `overwrite=false` given:

https://github.com/CompuIves/vscode/blob/master/src/vs/base/common/objects.ts#L85-L110

Reply:
Oh, and it does in recursively...

Reply:
Changed it to `mixin` to err on the safe side!